### ISSUE TITLE
Integer Number Selector up/down Arrors on Change

### ISSUE THREAD

**Issue by jtobago** [odoo/odoo#59063]
https://github.com/odoo/odoo/blob/e2a21ee5c87b1bd31b83cede62a1bdeb672080fc/addons/web/static/src/js/views/list/list_editable_renderer.js#L294

How to reproduce:
Add option on form _lines
 <field name="qty"  options='{"type": "number", "step": 1}'/>

Error on changing (clicking) up/down arrows:
Failed to execute 'setSelectionRange' on 'HTMLInputElement': The input element's type ('number') does not support selection.

Add following code to fix the issue:
                        //alert(currentWidget.formatType);
                        if (selectionRange && currentWidget.formatType !== 'integer') {

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - click on the blue door icon of the version row - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error (starting with the Odoo version). Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by jtobago** [comment]
Hello @pedrobaeza , no I have not used runbot, but I have confirmed the issue with odoo 12,13 and 14 code base. I'm just trying to help with error reporting. Where can I learn more on how to use runbot for better reporting?

**Comment by pedrobaeza** [comment]
It's just opening the URL, click the blue button and input admin/admin. The idea is to describe full steps to reproduce the error.

**Comment by jtobago** [comment]
ok added more information and screenshots.

**Comment by pedrobaeza** [comment]
But how do you get such arrows? It's not an option of the framework...

**Comment by jtobago** [comment]
on a view I've added a field option as follows:

`<field name="qty"  options='{"type": "number", "step": 1}'/>`

not really a widget I guess, but there are other references within odoo doc.

**Comment by pedrobaeza** [comment]
Which references? I would say that this is not a expected widget option and as such, it gives error. That's why we co-developed this module: https://github.com/OCA/web/tree/12.0/web_widget_numeric_step

**Comment by jtobago** [comment]
It's certainly an option and the only change needed to make it perfect is to the change/add the small code check as pointed. search odoo code for `options='{"type": "number"` and you will find references.
Would the OCA module you suggested work on many to one lines as well?

**Comment by pedrobaeza** [comment]
Yes, it should work. Anyway, why don't you propose that change as a pull request so that it may be included in core?
